### PR TITLE
Fixup run instructions

### PR DESCRIPTION
**Pull request by jappeace**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,5 +8,5 @@ Twitch/Discord Chat Bot that works for KGB ![monkaS](https://cdn.betterttv.net/e
 
 ```console
 $ cabal v2-build
-$ cabal v2-run secret.json database.db
+$ cabal v2-run kgbotka secret.json database.db
```
Comment (rexim) [inline README.md:11]: Is it not `exe:kgbotka`?

Comment (jappeace) [reply README.md:11]: this apparently works: 

image

Comment (rexim) [reply README.md:11]: Very well then

**Comment by rexim** [comment]
@jappeace looks good to me! Thanks for the contribution!

**Pull request merged**